iOS Client/Info.plist

```diff
@@ -20,6 +20,39 @@
 	<string>1</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSAppTransportSecurity</key>
+	<dict>
+		<key>NSExceptionDomains</key>
+		<dict>
+			<key>fireside.fm</key>
```

Comment:
I don't think this is necessary. Fireside supports `https` so we can just add that to the urls defined in our client above